src/hyp_map.rs

```diff
@@ -35,6 +35,9 @@ type UmodeElfRegionsVec = ArrayVec<UmodeElfRegion, MAX_UMODE_ELF_REGIONS>;
 // Hw map regions vector.
 type HwMapRegionsVec = ArrayVec<HwMapRegion, MAX_HW_MAP_REGIONS>;
 
+// Global reference to the Hypervisor Map.
```

Comment:
nit: Perhaps this commit message should start with `hyp_map`. Also would you consider adding why moving it is a good idea?

Reply:
I don't think the move is per se important in rust. It was just a clean up on the way code has evolved since the creation.